README.md

```diff
@@ -1,4 +1,4 @@
-Elgg [![Build Status](https://secure.travis-ci.org/Elgg/Elgg.svg?branch=1.x)](https://travis-ci.org/Elgg/Elgg) [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/Elgg/Elgg/badges/quality-score.png?s=1.x)](https://scrutinizer-ci.com/g/Elgg/Elgg/?branch=1.x) [![Scrutinizer Code Coverage](https://scrutinizer-ci.com/g/Elgg/Elgg/badges/coverage.png?b=1.x)](https://scrutinizer-ci.com/g/Elgg/Elgg/?branch=1.x) [![Read the docs build status](https://readthedocs.org/projects/elgg/badge/?version=1.x)](http://learn.elgg.org/en/1.x/)
+Elgg [![Build Status](https://secure.travis-ci.org/Elgg/Elgg.svg?branch=1.11)](https://travis-ci.org/Elgg/Elgg) [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/Elgg/Elgg/badges/quality-score.png?s=1.11)](https://scrutinizer-ci.com/g/Elgg/Elgg/?branch=1.11) [![Scrutinizer Code Coverage](https://scrutinizer-ci.com/g/Elgg/Elgg/badges/coverage.png?b=1.11)](https://scrutinizer-ci.com/g/Elgg/Elgg/?branch=1.11) [![Read the docs build status](https://readthedocs.org/projects/elgg/badge/?version=1.11)](http://learn.elgg.org/en/1.11/)
```

Comment:
I guess we can leave these pointing to 1.11 for now and update them again for 1.12 branch once we create it.